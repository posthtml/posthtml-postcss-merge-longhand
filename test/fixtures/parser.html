<div style="margin-top: 0; margin-bottom: 0 margin-left: auto; margin-right: auto">margin</div>
