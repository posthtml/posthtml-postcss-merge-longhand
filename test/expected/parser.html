<div style="margin: 0 auto">margin</div>
